libraries/Microsoft.Bot.Builder.LanguageGeneration/TemplateEngine.cs

```diff
@@ -154,5 +156,23 @@ public string Evaluate(string inlineStr, object scope = null, IGetMethod methodB
             var evaluator = new Evaluator(templates, methodBinder);
             return evaluator.EvaluateTemplate(fakeTemplateId, scope);
         }
+
+        private void CheckImportResolver(string id, ImportResolverDelegate importResolver)
+        {
+            // Currently if no resolver is passed into AddText(),
+            // the default fileResolver is used to resolve the imports.
+            // default fileResolver require resource id should be fullPath,
+            // so that it can resolve relative path based on this fullPath.
+            // But we didn't check the id provided with AddText is fullPath or not.
+            // So when id != fullPath, fileResolver won't work.
+            if (importResolver == null)
+            {
+                var importPath = ImportResolver.NormalizePath(id);
+                if (!Path.IsPathRooted(importPath))
+                {
+                    throw new Exception("[Error] ImportResolver can't be empty when using AddText unless id is full path string");
```

Comment:
You should say, `id must be full path when importResolver is null`, the issue is not import can't be empty, the issue is id should be full path